keshflou/javascript/script.js

```diff
@@ -1,3 +1,4 @@
+//slick sliders
 $(document).ready(function(){
 
```

Comment:
@safronman it is best practice to put everything inside $.ready function. please move lines 25+ inside here.